STUDIO/org.openl.security.standalone/resources/db/flyway/oracle/V9__ACL.sql

```diff
@@ -0,0 +1,40 @@
+CREATE TABLE acl_sid (
+	id NUMBER(38) NOT NULL PRIMARY KEY,
+	principal NUMBER(1) NOT NULL CHECK (principal in (0, 1)),
+	sid NVARCHAR2(255) NOT NULL,
+	CONSTRAINT unique_acl_sid UNIQUE (sid, principal)
+);
+
+CREATE TABLE acl_class (
+	id NUMBER(38) NOT NULL PRIMARY KEY,
+	class NVARCHAR2(2000) NOT NULL,
+	class_id_type NVARCHAR2(2000),
+	CONSTRAINT uk_acl_class UNIQUE (class)
+);
+
+CREATE TABLE acl_object_identity (
+	id NUMBER(38) NOT NULL PRIMARY KEY,
+	object_id_class NUMBER(38) NOT NULL,
+	object_id_identity NVARCHAR2(2000) NOT NULL,
+	parent_object NUMBER(38),
+	owner_sid NUMBER(38),
+	entries_inheriting NUMBER(1) NOT NULL CHECK (entries_inheriting in (0, 1)),
+	CONSTRAINT uk_acl_object_identity UNIQUE (object_id_class, object_id_identity),
+	CONSTRAINT fk_acl_object_identity_parent FOREIGN KEY (parent_object) REFERENCES acl_object_identity (id),
+	CONSTRAINT fk_acl_object_identity_class FOREIGN KEY (object_id_class) REFERENCES acl_class (id),
+	CONSTRAINT fk_acl_object_identity_owner FOREIGN KEY (owner_sid) REFERENCES acl_sid (id)
+);
+
+CREATE TABLE acl_entry (
+	id NUMBER(38) NOT NULL PRIMARY KEY,
```

Comment:
Have you tried to use 
`${bigint}` instead of `NUMBER(38)`
`${varchar}` instead of `NVARCHAR2`
`${boolean}` instead of `NUMBER(1)`

  to generalize all scripts?